Sources/CarPlay/Templates/HAEntity+CarPlay.swift

```diff
@@ -60,6 +60,8 @@ extension HAEntity {
             default:
                 break
             }
+        case .sensor:
```

Comment:
You can add `.sensor` withint the `.none` case, since on press we dont have any action for the sensor

Reply:
Done